hackathon_site/dashboard/frontend/src/components/user/AcknowledgementForm/AcknowledgementForm.js

```diff
@@ -0,0 +1,180 @@
+    const firstHalf =
+        "IN SIGNING THIS RELEASE, I ACKNOWLEDGE AND REPRESENT THAT I have read the foregoing ";
+    const secondHalf =
+        ", understand it and sign it voluntarily as my own free act and deed; no oral representations, statements, or inducements, apart from the foregoing written agreement, have been made; I am at least eighteen (18) years of age and fully competent; and I execute this Release for full, adequate and complete consideration fully intending to be bound by same.";
+
+    return (
+        // <form className={styles.acknowledge} onSubmit={handleContinue}>
+        //     <FormControlLabel
+        //         control={<Checkbox name="firstCheck" />}
+        //         label={checkBox1}
+        //     />
+        //     <FormControlLabel
+        //         control={<Checkbox name="secondCheck" />}
+        //         label={checkBox2}
+        //     />
+        //     <FormControlLabel
+        //         control={<Checkbox name="thirdCheck" />}
+        //         label={checkBox3}
+        //     />
+        //     <FormControlLabel
+        //         control={<Checkbox name="forthCheck" />}
+        //         label={checkBox4}
+        //     />
+        //     <Typography variant="body1" className={styles.acknowledgeText}>
+        //         {firstHalf}
+        //         <Link href="#" onClick={openDoc}>
+        //             Waiver of Liability and Hold Harmless Agreement
+        //         </Link>
+        //         {secondHalf}
+        //     </Typography>
+        //     <TextField
+        //         label="e-signature"
+        //         className={styles.acknowledgeSignature}
+        //         name="signature"
+        //     />
+        //     <Button
+        //         type="submit"
+        //         className={styles.acknowledgeButton}
+        //         variant="contained"
+        //         color="primary"
+        //         onClick={handleContinue}
+        //     >
+        //         Continue
+        //     </Button>
+        // </form>
+        <div className={styles.acknowledge} onSubmit={handleContinue}>
+            <FormikCheckBox
+                name="agreement"
+                value={checkBox1}
+                className={styles.acknowledgeCheckbox}
+            />
+            <FormikCheckBox
+                name="agreement"
+                value={checkBox2}
+                className={styles.acknowledgeCheckbox}
+            />
+            <FormikCheckBox
+                name="agreement"
+                value={checkBox3}
+                className={styles.acknowledgeCheckbox}
+            />
+            <FormikCheckBox
+                name="agreement"
+                value={checkBox4}
+                className={styles.acknowledgeCheckbox}
+            />
+            <Typography variant="body1" className={styles.acknowledgeText}>
+                {firstHalf}
+                <Link href="#" onClick={openDoc}>
```

Comment:
@lisa-sa-li can you confirm, but I think this can just be a plain old link to some document on our website? Or is this supposed to pop up a react component that contains the waiver (pdf on our site would probably be the easiest)

Reply:
Yup, just the `<a>` tag is enough